support-frontend/migration/README.md

```diff
@@ -0,0 +1,59 @@
+# Typescript Migration
+
+This folder contains all of the configuration files- either brand new or changed from our existing files- scripts and Github workflows required for the migration of the client-side code to Typescript.
+
+## Performing the migration
+
+### Setting up configuration files
+
+All files should replace any existing versions of the same file in their destination folder.
+
+- Files in /scripts belong in support-frontend/support-frontend/scripts
+- Files in /workflows belong in support-frontend/.github/workflows
+- Files in /.storybook belong in support-frontend/support-frontend/.storybook
+- All other files belong in support-frontend/support-frontend
+
+### Running the migration script
+
+1. Make copies of the /stories and /assets folders
+2. Run `yarn` to install all dependencies in the new package.json
+3. Run `node scripts/ts-migrate.js --dir=assets` to migrate the assets directory (this may take a while!)
+4. Run `node scripts/ts-migrate.js --dir=stories` to migrate the stories directory
+
+### Post-migration manual fixes
+
+There are a few manual syntax fixes that need to be applied. Search and replace in both /assets and /stories for the following:
+
+- Replace “import type” with “import”
+- Replace “, type “ with “, “
+- Replace “{ type” with “{“
```

Comment:
Discussed already but just for posterity: we might not need these, and the last one affects a few comments in the code